<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestmentsWindow</class>
 <widget class="QDialog" name="InvestmentsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="loggedUserLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>loggedUserLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="invesmentsList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>661</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>List of your investments:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add new investment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete investment</string>
   </property>
  </widget>
  <widget class="QLabel" name="usdLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>USD</string>
   </property>
  </widget>
  <widget class="QLabel" name="jpyLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Yen</string>
   </property>
  </widget>
  <widget class="QLabel" name="gbpLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pound</string>
   </property>
  </widget>
  <widget class="QLabel" name="czkLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Czech koruna</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Russian ruble</string>
   </property>
  </widget>
  <widget class="QLabel" name="eurLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Euro</string>
   </property>
  </widget>
  <widget class="QLabel" name="nokLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Norwegian krone</string>
   </property>
  </widget>
  <widget class="QLabel" name="dkkLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Danish krone</string>
   </property>
  </widget>
  <widget class="QLabel" name="brlLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Brazilian real</string>
   </property>
  </widget>
  <widget class="QLabel" name="audLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Australian dolar</string>
   </property>
  </widget>
  <widget class="QLabel" name="usdAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="eurAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="jpyAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="gbpAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="czkAmount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="audAmount">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="brlAmount">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="dkkAmount">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="nokAmount">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="rubAmount">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>490</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify investment</string>
   </property>
  </widget>
  <widget class="QLabel" name="valueInfoLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Global value of investments (current amount of PLN only): </string>
   </property>
  </widget>
  <widget class="QLabel" name="plnAmount">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
